kvmagent/kvmagent/plugins/ovn.py

```
@@ -163,72 +163,134 @@ def start_ovn_service(self, req):
+                        err, pci = vsctl.getTableAttr("Interface", nicName, "options:dpdk-devargs")
+                        if err or pci != pciAddress:
+                            needAddUplink = True
+                            break
+                        err1, nicRxQueueNum = vsctl.getNicRxQueueNumConfig(nicName)
+                        err2, nicRxQueueDescNum = vsctl.getNicRxQueueDescNumConfig(nicName)
+                        if err1 or err2 or nicRxQueueNum != cmd.nicRxQueueNumber or nicRxQueueDescNum != cmd.nicRxQueueDescNumber:
+                            r = vsctl.setNicRxQueueConfig(nicName, cmd.nicRxQueueNumber, cmd.nicRxQueueDescNumber)
+                            if r != 0:
+                                msg = "set ovs dpdk rx queue config failed!"
+                                return self._logRestoreNicDriverMakeRsp(rsp, msg, cmd)
+                        needAddUplink = False
```

Pylint (3.3.7)

[refactor] Too many nested blocks (6/5)

(R1702)